src/Services/EntitiesInformation/Explorer.php

```diff
@@ -196,18 +270,18 @@ protected function loadMethodAnnotationForMethodWithNameAndEntityWithClassName($
     /**
      * @param string $propertyName
      * @param string $entityPropertiesClassName
-     * @return array
+     * @return type
      */
     protected function loadPropertyAnnotationForPropertyWithNameAndEntityPropertiesWithClassName($propertyName, $entityPropertiesClassName) {
-        $annotations = array();
+        $annotations = [];
 
         foreach($this->ancestorsForClassWithName($entityPropertiesClassName) as $class) {
             if (!$class::getReflection()->hasProperty($propertyName)) continue;
-            $classAnnotations = array();
+            $classAnnotations = [];
```

Comment:
arrays should be unified... brackets are used here, but ive seen lot of array definition with array keyword